tests/installation/bootloader.pm

```diff
@@ -28,6 +29,18 @@ sub run() {
     registration_bootloader_params(utils::VERY_SLOW_TYPING_SPEED);
     select_bootmenu_language;
     select_bootmenu_video_mode;
+
+    # CaaSP specific boot dependencies
```

Comment:
I think I would place it into bootmenu_default_params as it does nothing, just wait, right ?
and use  with it return 3 as you don't need any other options